res/translations/de.json

```diff
@@ -0,0 +1,231 @@
+    "Content type": "Inhaltstyp",
+    "Could not reach server": "Server nicht erreichbar",
+    "Create new playlist": "Neue Playlist erstellen",
+    "day": "Tag",
+    "days": "Tage",
+    "Delete Playlist": "Playlist löschen",
+    "Demo": "Demo",
+    "Description": "Beschreibung",
+    "Disable server transcoding": "Server-Transkodierung deaktivieren",
+    "Disc number": "Disk Nummer",
+    "Discography": "Diskographie",
+    "discs": "Disks",
+    "DJ-Mix": "DJ-Mix",
+    "Download": "Download",
+    "Download completed": "Download abgeschlossen",
+    "Duration": "Dauer",
+    "Edit": "Bearbeiten",
+    "Edit Playlist": "Playlist bearbeiten",
+    "Edit server": "Server bearbeiten",
+    "Enable system tray": "Taskleiste aktivieren",
+    "Enabled": "Aktiviert",
+    "Enter": "Eingeben",
+    "EP": "EP",
+    "Equalizer": "Equalizer",
+    "Exclusive mode": "Exklusiver Modus",
+    "Favorites": "Favoriten",
+    "Field Recording": "Field Recording",
+    "File path": "Dateipfad",
+    "File size": "Dateigröße",
+    "Filter albums": "Alben filtern",
+    "Filter genres": "Genres filtern",
+    "Forward": "Vorwärts",
+    "Frequently Played": "Häufig gespielt",
+    "General": "Allgemein",
+    "Genre": "Genre",
+    "Genres": "Genres",
+    "Github page": "Github-Seite",
+    "Go to release page": "Zur Release-Seite gehen",
+    "Hide": "Ausblenden",
+    "Home": "Startseite",
+    "Home Page": "Startseite",
+    "hr": "Std.",
+    "hrs": "Std.",
+    "Internet Radio Stations": "Internet-Radiosender",
+    "Interview": "Interview",
+    "Is favorite": "Ist Favorit",
+    "Is not favorite": "Ist kein Favorit",
+    "Last played": "Zuletzt gespielt",
+    "Live": "Live",
+    "Locally": "Lokal",
+    "Log Out": "Abmelden",
+    "Login to Server": "Beim Server anmelden",
+    "Lyrics": "Songtext",
+    "Lyrics not available": "Songtext nicht verfügbar",
+    "Menu": "Menü",
+    "min": "Min.",
+    "minutes of track have been played": "Minuten des Titels abgespielt wurden",
+    "Mixtape": "Mixtape",
+    "Mute": "Stumm",
+    "My Server": "Mein Server",
+    "Name": "Name",
+    "Name (A-Z)": "Name (A-Z)",
+    "Next": "Nächster",
+    "Nickname": "Spitzname",
+    "Now Playing": "Aktuelle Wiedergabe",
+    "No new version found": "Keine neue Version gefunden",
+    "No radio stations available": "Keine Radiosender verfügbar",
+    "None": "Keine",
+    "none": "keine",
+    "none selected": "keine ausgewählt",
+    "OK": "OK",
+    "optional": "optional",
+    "or when": "oder wenn",
+    "Password": "Passwort",
+    "Pause": "Pause",
+    "Paused": "Pausiert",
+    "Peak Meter": "Peak Meter",
+    "percent of track is played": "Prozent des Titels abgespielt",
+    "Play": "Abspielen",
+    "Play Artist Radio": "Künstler-Radio abspielen",
+    "Play count": "Anzahl der Wiedergaben",
+    "Play Discography": "Diskographie abspielen",
+    "Play next": "Als nächstes abspielen",
+    "Play Queue": "Wiedergabeliste abspielen",
```

Comment:
This is one that a lot of translators have been incorrect on initially (I wish JSON could have comments!) -- 

"Play Queue" is used as a noun, as the title for the queue popup and the tab that shows the queue on the Now Playing page. I notice the translation here looks to be the same pattern as "Play Discography", etc, so I'm assuming you translated it as an action (eg if it were a button to press to play the queue)